topi/python/topi/generic/nn.py

```diff
@@ -229,3 +229,54 @@ def schedule_binary_dense(outs):
         The computation schedule for the op.
     """
     return _default_schedule(outs, False)
+
+@tvm.target.generic_func
+def schedule_shortcut(outs):
+    """Schedule for shortcut
+
+    Parameters
```

Comment:
Need to be careful that the current schedules do not works for GPU. We need to include a GPU schedule

Reply:
Now we are working on mali gpu, once its done, will push the code for gpu.

Reply:
The default schedule does not work for GPU. We need to at least make sure that the schedule runs on GPU devices without crashing them, and add test cases to guard these before we merge.